frontend/src/components/Button.cy.tsx

```diff
@@ -0,0 +1,27 @@
+import Button from "./Button";
```

Comment:
it seems like example components, lets our own real components!

Reply:
Changed - now the tet is using the project button